Python/Product/DebuggerHelper/DebuggerHelper.vcxproj

```diff
@@ -60,6 +60,7 @@
   </PropertyGroup>
   <PropertyGroup Label="Configuration" Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <PlatformToolset>v143</PlatformToolset>
+    <SpectreMitigation>Spectre</SpectreMitigation>
```

Comment:
This one is only set for `Debug|x64` per property group condition - it needs to be defined for other configurations, too, or better yet, in an unconditional `<PropertyGroup>` so that it applies to all of them.

Reply:
(and ditto for all the other properties)

Reply:
Ahh you are right. I only added them for Debug, and for Debug there is only a single platform for each of these projects. But for release, there is both x64 and x86. I will fix this.

Reply:
Ok this is fixed in the latest PR update.